remote-method/index.js

```diff
@@ -91,14 +91,16 @@ module.exports = yeoman.generators.Base.extend({
         message: 'Enter the remote method name:',
         required: true,
         default: name,
-        validate: checkPropertyName
+        validate: checkRemoteMethodName
       },
       {
         name: 'isStatic',
         message: 'Is Static?',
-        required:true,
+        required: false,
         type: 'confirm',
-        default: true
+        default: function(answers) {
+          return !answers.methodName.match(/^prototype\.(.*)$/);
```

Comment:
Ah, I see, the prefix is removed here: https://github.com/strongloop/generator-loopback/pull/173/files#diff-1162a393eb1e8979663fc397b9fbacbeR113 

✅